dashboard/lib/queries/lti.rb

```diff
@@ -29,7 +29,7 @@ def self.get_course_from_context(lti_integration_id, context_id)
   end
 
   def self.get_lti_course_from_section_code(section_code)
-    Section.find_by(code: section_code).lti_course
+    Section.find_by(code: section_code)&.lti_course
```

Comment:
Without the &, this throws an error that results in a 500 when we call it from the controller. Figured a nil return value would be more appropriate.